projects/pcapplusplus/default.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+close_fd_mask=3
```

Comment:
@zlowram @seladb - probably should fix this later, this causes any stdout/stderr from program to be not visible, so you will not catch asserts (if any). fine for now.